### PR TITLE
NAS-110524 / 21.08 / Fix relation issue of entity wizard

### PR DESCRIPTION
**Pull request by AlexKarpov98**

### CODE REVIEW

File: src/app/pages/common/entity/entity-wizard/entity-wizard.component.ts [hunk 1]
```diff
@@ -108,11 +108,10 @@ export class EntityWizardComponent implements OnInit {
     });
 
     for (const i in this.conf.wizardConfig) {
+      const formGroup = < FormGroup > this.formArray.get(i);
```
Comment (undsoft) [inline src/app/pages/common/entity/entity-wizard/entity-wizard.component.ts:111]: [Please prefer](https://google.github.io/styleguide/tsguide.html#type-assertions-syntax) `as FormGroup` for casting.

**Review by undsoft — APPROVED** [review]
I've tested:
- GUI settings in handbrake
- Host path in nextcloud
- Host path in plex https://jira.ixsystems.com/browse/NAS-110805
These appear to be working okay.

Снимок экрана 2021-05-29 в 10 42 31

**Comment by undsoft** [comment]
Also, please retarget to `release/21.06-BETA.1`, so that it gets included in next release.

**Review by Ornias1993 — APPROVED** [review]
Thanks @AlexKarpov98  for taking this up so quickly :)
and
Thanks @undsoft for verifying it thoroughly. 

This is indeed exactly what we need to get the Apps into running shape for 21.06-BETA 👍

**Pull request merged**